### PR TITLE
Add basic UI

### PR DESCRIPTION
**Pull request by PeteUgwu**
## Added basic UI

- Created an app.rb file that will serve as the console app entry-point, and implemented methods as required.
- Added a method called main in main.rb file, which is invoked at the end of the file to provide basic UIs to users.
- Followed GitFlow

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+  def create_student(age, name, parent_permission)
+    classroom = 'Null'
+    @people << { type: 'Student', data: Student.new(age, name, classroom, parent_permission) }
+  end
+
+  def create_teacher(age, name, spec)
+    @people << { type: 'Teacher', data: Teacher.new(age, spec, name) }
+  end
+
```
Comment (DeliceLydia) [inline app.rb:21]: - [ ] I am not able to create a `student` and `teacher` please address it
Screen Shot 2023-03-17 at 18 47 43

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @PeteUgwu, 

Good job so far! There are some issues that you still need to work on to go to the next project but 
you are almost there!


## Highlights
- Good PR description
- No linter errors

## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb (hunk 1 above)
Comment (PeteUgwu) [reply app.rb:21]: Thanks. It has been fixed

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @PeteUgwu , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- [ ] All linters passed

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**